update

```diff
@@ -48,8 +48,13 @@ dirs.forEach(function (dir) {
         return pars
       })
       .sort(function (a, b) {
-        // NOTE: a vs. b (the order is important), because we want oldest first in the list
-        return semver.compare(a.longVersion, b.longVersion)
+        // NOTE: a vs. b (the order is important), because we want oldest first in the list.
```

Comment:
This isn't completely true. Only the `.json` list is in descending order. Should I fix the comment or reverse the lists?

Reply:
Maybe just fix the comment.

Reply:
Ok. Changing it now.

Reply:
Done. Actually the comment is accurate because it refers only to `parsedList` rather than all the `list` files. I edited it to make it clear.